### PR TITLE
[CARBONDATA-234]wrong message getting displayed while compaction.

### PR DESCRIPTION
**Pull request by ravikiran23**
Why raise this pr?

for each compaction the error message "compaction request file can not be deleted " is printing even though it has been deleted from the file system.

How to resolve?

The return of the boolean value is missing in the API deleteCompactionRequiredFile(). Need to return the boolean value correctly for getting the delete status.

### CODE REVIEW

File: integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java [hunk 1]
```diff
@@ -210,10 +211,14 @@ public static boolean deleteCompactionRequiredFile(String metaFolderPath,
       if (FileFactory.isFileExist(statusFile, FileFactory.getFileType(statusFile))) {
         if (FileFactory.getCarbonFile(statusFile, FileFactory.getFileType(statusFile)).delete()) {
           LOGGER.info("Deleted the compaction request file " + statusFile);
+          return true;
         } else {
           LOGGER.error("Unable to delete the compaction request file " + statusFile);
         }
       }
+      else {
```
Comment (jackylk) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: move one line up

File: integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java [hunk 2]
```diff
@@ -246,6 +251,9 @@ public static boolean createCompactionRequiredFile(String metaFolderPath,
           return false;
         }
       }
+      else {
```
Comment (jackylk) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: move one line up

File: integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java (hunk 1 above)
Comment (ravikiran23) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: done

File: integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java (hunk 2 above)
Comment (ravikiran23) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: done

File: integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java [hunk 3]
```diff
@@ -210,9 +210,12 @@ public static boolean deleteCompactionRequiredFile(String metaFolderPath,
       if (FileFactory.isFileExist(statusFile, FileFactory.getFileType(statusFile))) {
```
Comment (kumarvishal09) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: change statusFile to compactionRequestFile

Comment (ravikiran23) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: done

Comment (ravikiran23) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/225/

**Comment by gvramana** [comment]
LGTM

**Pull request closed without merging**